cmd/lakectl/cmd/bisect_start.go

```diff
@@ -13,8 +13,8 @@ var bisectStartCmd = &cobra.Command{
 	Short: "Start a bisect session",
 	Args:  cobra.ExactArgs(bisectStartCmdArgs),
 	Run: func(cmd *cobra.Command, args []string) {
-		badURI := MustParseRefURI("bad", args[0])
-		goodURI := MustParseRefURI("good", args[1])
+		badURI := MustParseRefURI("bad ref", args[0])
+		goodURI := MustParseRefURI("good ref", args[1])
```

Comment:
Elsewhere we say something more like
```suggestion
		badURI := MustParseRefURI("bad ref URI", args[0])
		goodURI := MustParseRefURI("good ref URI", args[1])
```